cumulusci/tasks/tests/test_metadeploy.py

```diff
@@ -189,10 +207,48 @@ def test_run_task(self):
                         "checks": [],
                     },
                 },
+                {
+                    "name": "util_sleep",
```

Comment:
Sleep was just a handy task without side effects to use for the test. It didn't matter what task but I needed to add a step with a step-level preflight check (as opposed to plan-level) to make sure that this line gets test coverage: https://github.com/SFDO-Tooling/CumulusCI/pull/1577/files#diff-0b667355598f51590b2539b31b3b3ca3R169 (and it's a good thing I did, because it revealed that I was looking for the checks in the wrong place)